### PR TITLE
hostapd: new recipe

### PR DESCRIPTION
**Pull request by hundeboll**

### CODE REVIEW

File: recipes/hostapd/hostapd.inc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+DESCRIPTION = "User space daemon for extended IEEE 802.11 management"
+HOMEPAGE = "http://hostap.epitest.fi"
+LICENSE = "GPLv2 BSD"
+DEPENDS = "libnl-3 libnl-genl-3 openssl librt"
```
Comment (kimrhh) [inline recipes/hostapd/hostapd.inc]: no RDEPENDS at all? hmm :)

File: recipes/hostapd/hostapd.inc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+do_configure() {
+    install -m 0644 ${SRCDIR}/defconfig ${S}/.config
+}
+
+CFLAGS = "-MMD -O2 -Wall -g -I${MACHINE_SYSROOT}${includedir}/libnl3"
+do_compile() {
+    oe_runmake
+}
+
```
Comment (kimrhh) [inline recipes/hostapd/hostapd.inc]: this is the default, should not need to specify it, or am I missing something?

File: recipes/hostapd/hostapd.inc (hunk 1 above)
Comment (hundeboll) [inline recipes/hostapd/hostapd.inc]: OK, let me do a packageqa

**Comment by kimrhh** [comment]
merged, thanks

**Pull request merged**